python/lsst/sconsUtils/scripts.py

```diff
@@ -481,9 +507,31 @@ def s(l):
             swigMods.extend(
                 state.env.SwigLoadableModule("_" + name, src, LIBS=state.env.getLibs("main python"))
             )
+
+        # Warn about insisting that a test in pySingles starts with test_ and therefore
+        # might be automatically discovered by pytest. This warning could only be issued
```

Comment:
I was attempt to call out that "could" either is in past tense or implies a level of uncertainty.  It would be better to be definitive and say either "can only be" or "should only be".